api_views/users.py

```
@@ -12,6 +12,15 @@
 def error_message_helper(msg):
     return '{ "status": "fail", "message": "' + msg + '"}'
 
+def error_message_helper1(msg):
+    try:
+        jsonschema.validate(msg, msg)
+    except:
+        return {"status": 200, "message": "' + msg + '"}
```

Bare except clause masks validation errors - should catch JsonSchemaValidationError